### DISCUSSION TITLE
Subtitle Edit 4.0.0 issue

### DISCUSSION THREAD

**Discussion by likangkao** [SubtitleEdit/subtitleedit#7333] [General]
“I am currently using Subtitle Edit 4.0.0 with Purfview’s Faster-Whisper. Every time I use it, a pop-up window appears saying ‘Download Purfview’s Faster-Whisper (Update)’. However, no matter how many times I click the [Yes] button to update, the pop-up window reappears when I use Purfview’s Faster-Whisper again.

**Comment by niksedk** [comment]
Sorry, that's a bug... will be fixed in next update.
You can try the portable beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.0/SubtitleEditBeta.zip

**Reply by PreetM07** [reply]
I second this issue.

Another thing I noticed is, if the length is too big, the generation is interrupted randomly somewhere.
And surely while processing it is heating up like h****!
